html/pfappserver/root/src/views/Configuration/connectionProfiles/_components/TheForm.vue

```diff
@@ -138,6 +138,11 @@
           :text="$i18n.t('Under most circumstances we can redirect the user to the URL he originally intended to visit. However, you may prefer to force the captive portal to redirect the user to the redirection URL.')"
         />
 
+        <form-group-show-manage-devices-on-max-nodes namespace="show_manage_devices_on_max_nodes"
+          :column-label="$i18n.t('Show manage devices options when user reaches max nodes')"
+          :text="$i18n.t('When the user reaches the maximum amount of nodes for his username, enabling this will display an option in the portal to access the status page to manage/unregister devices.')"
```

Comment:
When the user reaches the maximum amount of nodes for his username => When the user reaches the maximum amount of registered nodes associated with his or her username

Reply:
What about non binary?